src/compositemark.ts

```diff
@@ -0,0 +1,64 @@
+import {Encoding} from './encoding';
```

Comment:
No -- it's not single instance.  

This way we can add experimental plugins for box plot / error bar / confidence internal that we need for CompassQL and Voyager without adding them to the main API. 

(In fact, in a future PR, I am leaning toward moving ERRORBAR out of here.)